js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,13 @@
+function lastIndexOf(...args) {
+  let searchedValue = args[0];
+  let list = args[1];
+  for (let i = list.length; i >= 0; i--) {
+    if (list[i] === searchedValue) {
+      return i;
+    }
+  }
+  return -1;
```

Comment:
returning -1 here when value is not present. We can do something like this on top ` if (list.indexOf(searchedValue) === -1) return -1;
`